mocks/accept.go

```diff
@@ -0,0 +1,10 @@
+package mocks
```

Comment:
Smitty is moving this into the thirdparty directory. Although it is fake data, it will actually be returned "for real" as we're faking the thirdparty backend.